src/authx/auth.py

```diff
@@ -703,7 +708,7 @@ def remove_provider_from_tyk_api(api_id, issuer, policy_id=TYK_POLICY_ID):
     response = requests.request("GET", url, headers=headers)
     if response.status_code == 200:
         api_json = response.json()
-        new_providers = []
+        new_providers = [api_json['openid_options']['providers'].pop()]
```

Comment:
This currently ensures that the last-added provider (i.e. the most recently added provider) is kept in `new_providers`; it should probably instead be the first-added provider (i.e. the one corresponding to the local installation).
```suggestion
        new_providers = [api_json['openid_options']['providers'][0]]
```

Reply:
Actually it should be pop(0)